SixTrack/sixtrack.s

```diff
@@ -18492,6 +18828,16 @@ c$$$         endif
          !Moved from daten()
          crabph4(ix)=el(ix)
          el(ix)=0d0
+!--Wire
```

Comment:
OK, I was just a bit confused about what this code was doing in initialize_element.